index.html

```diff
@@ -82,6 +82,73 @@ <h1>I’m Azaria<br>
                 </ul>
             </div>
         </section>
+        <section class="about body flex-column bg-white w-90 b-rounded-right p-15">
+            <div class="title">
+                <h1>About<br> Myself</h1>
+            </div>
+            <div class="content">
+                <p>
+                    Hello I’m a software developer! I can help you build a product , feature or website 
+                    Look through some of my work and experience! If you like what you see and have a project
+                     you need coded, don’t hestiate to contact me.
+                </p>
+
+            </div>
+            <div class="connection">
+                <a href="#">LET’S CONNECT</a>
+            </div>
+            <div class="social-network">
+                <ul>
+                    <li>
+                        <a href="#">
+                            <img src="./images/twitter.svg" alt="twitter" />
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                            <img src="./images/Linkedin_icon.svg" alt="linkedin" />
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                            <img src="./images/angellist.svg" alt="angellist" />
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                            <img src="./images/github.svg" alt="github" />
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                            <img src="./images/medium.svg" alt="medium" />
+                        </a>
+                    </li>
+                </ul>
+            </div>
+            <div class="m-t-5 m-b-5">
+                <button class="btn p-5">
+                    Get my resume
+                </button>
+            </div>
+            <div class="categories m-t-5">
+                <ul class="p-5">
+                    <li class="w-100">
+                       <span class="text">Langguages</span> 
+                       <span class="icon" ><img src="./images/arrow_1.png" alt="arrow" /></span>
+                    </li>
+                    <li class="w-100">
+                        <span class="text">Frameworks</span>
+                        <span class="icon" ><img src="./images/arrow_1.png" alt="arrow" /></span>
+                    </li>
+                    <li class="w-100">
+                        <span class="text">Skills</span>
+                        <span class="icon" ><img src="./images/arrow_1.png" alt="arrow" /></span>
+                    </li>
+                </ul>
+
+            </div>
+        </section>
     </div>
 </body>
```

Comment:
- [ ] It would be better if we maintain a professional README file by adding the following:
  - [ ] Instructions about how to clone and run the app locally
  - [ ] Provide social media handles
It would be better if we use the recommended README [template](https://github.com/microverseinc/readme-template)